src/components/SimpleLocationRouteInputForm/SimpleLocationRouteInputForm.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import LocationInput from '../LocationInput/LocationInput';
+
+const SimpleLocationRouteInputForm = ({ onStartingPointChange, onDestinationChange }) => {
+  return (
+    <div>
+      <h2>Simple Location Route Input Form</h2>
```

Comment:
this header seems odd - while it might not be the case yet, the form should at some point become self-explanatory without the need of a header. i'd remove it for now

Reply:
removed